### DISCUSSION TITLE
ImagePicker选择照片后，删除时怎么增加是否要删除的提示

### DISCUSSION THREAD

**Discussion by linksay** [TaleLin/lin-ui#1262] [求助] [answered]
ImagePicker选择照片后，删除时怎么增加是否要删除的提示

**Comment by juzi214032** [comment]
暂时不支持增加删除提示，后续我们会考虑一下这个能否实现

**Comment by juzi214032** [comment, marked as answer]
0.9.4 版本已支持该场景
使用 remove 属性禁用删除图片，并监听 linremovetap 事件，最后调用外部函数 linRemoveImage 即可
